setup.py

```diff
@@ -227,21 +235,39 @@ def ctresrc(self):
 get = Downloader()
 
 _travis_build = os.environ.get("TRAVIS_BUILD")
-
+libraries = None
+cmdclass = {}
+if ctre_rpi:
+    ext_modules = [
+        Extension(
+            "ctre._impl.ctre_cpp",
+            ["ctre/_impl/ctre_cpp.cpp"],
+            include_dirs=[
+                # Path to pybind11 headers
+                get_pybind_include(),
+                get_pybind_include(user=True),
+                get.ctresrc,
+            ],
+            libraries=libraries,
```

Comment:
Aren't you missing the CCI library here?

Reply:
Yeah, I thought it was part and parcel with the HAL and as far as I understand there's no HAL for raspberrypi.  Okay, I've committed a change where it includes the CCI but not the HAL if it's on raspberrypi.

Reply:
Well, you _do_ want to be able to use the CTRE libraries. We only provide bindings; the underlying driver implementation is from CTRE.

Reply:
Yeah, I realize that now :-).

I'm not trying to pull in the whole FRC framework, just trying to code a fairly straightforward python script that uses a talon SRX to control a couple motors. 

I'm not sure though, I'm looking at what I think is the output of this build process, ./robotpyt-ctre/dist/robotpy-ctre-2019.3.0.tar.gz and unpacking it, I'm seeing stuff importing hal.  

Since AFAIK there is no hal for raspberrypi, that sounds like it's going to be a problem.

Is there a suitable place I could post followup questions on the larger topic?  

I guess the github issue:  https://github.com/robotpy/robotpy-ctre/issues/106

Is there a good place to post more general questions?

Reply:
We hang out on gitter sometimes. https://gitter.im/robotpy/robotpy-wpilib